pom.xml

```diff
@@ -2854,6 +2854,14 @@
           </execution>
         </executions>
       </plugin>
+      <!-- Couple of dependencies are coming in bundle format. If this removed build will fail. -->
```

Comment:
Removed my m2 repo and restarted compilation. In the meantime I've excluded a dependency and want to double check if the problem still exists + going through again to see what kind of exception has been thrown. AFAIK missing dependency or something like this...

Reply:
So the exception still comes:
```
[ERROR] Failed to execute goal on project spark-sql-kafka-0-10_2.12: Could not resolve dependencies for project org.apache.spark:spark-sql-kafka-0-10_2.12:jar:3.0.0-SNAPSHOT: Could not find artifact org.apache.directory.jdbm:apacheds-jdbm1:bundle:2.0.0-M2 in cloudera-mirror (http://maven.jenkins.cloudera.com:8081/artifactory/cloudera-mirrors) -> [Help 1]
```

Reply:
Added meaningful explanation + removed `<inherited>true</inherited>` since it's the default + removed the plugin from `kafka-0-10-sql` because if the plugin is in `<plugins>` section then it's available in children.

Reply:
Tested with maven and sbt as well.